Ginger/GingerCoreCommon/Repository/ApplicationModelLib/POMModelLib/ApplicationPOMModel.cs

```diff
@@ -55,6 +85,21 @@ public ApplicationPOMModel()
             }
         }
 
+        string mMappedBusinessFlow;
+        [IsSerializedForLocalRepository]
+        public string MappedBusinessFlow 
```

Comment:
MappedBusinessFlow type should be RepositoryItemKey and not String

Reply:
done